README.md

```diff
@@ -52,6 +52,41 @@ android {
   }
 ```
 
+4. Add this to /android/src/main/res/xml/network_security_config.xml
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<network-security-config>
+  <domain-config cleartextTrafficPermitted="true">
+    <domain includeSubdomains="true">ocsp.digicert.com</domain>
+    <domain includeSubdomains="true">crl3.digicert.com</domain>
+    <domain includeSubdomains="true">crl4.digicert.com</domain>
+    <domain includeSubdomains="true">crl.godaddy.com</domain>
+    <domain includeSubdomains="true">certificates.godaddy.com</domain>
+    <domain includeSubdomains="true">crl.starfieldtech.com</domain>
+    <domain includeSubdomains="true">certificates.starfieldtech.com</domain>
+    <domain includeSubdomains="true">ocsp.godaddy.com</domain>
+    <domain includeSubdomains="true">ocsp.starfieldtech.com</domain>
+  </domain-config>
+</network-security-config>
```

Comment:
It is not clear why these exceptions are needed for release. Could you provide an explanation and/or some link?

Reply:
Hi @mieszko4, sure I got those from the changelog, you can take a look here.

https://marketplace.zoom.us/docs/guides/stay-up-to-date/changelog/